shared/profile/pgp/finished-generating-pgp.desktop.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import React, {Component} from 'react'
+import {StandardScreen, Box, Text, Button, Icon, Checkbox} from '../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../styles/style-guide'
+import type {Props} from './finished-generating-pgp'
+
+type State = {
+  shouldStoreKeyOnServer: boolean,
+}
+
+class FinishedGeneratedPgp extends Component<void, Props, State> {
+  state: State;
+
+  constructor (props: Props) {
+    super(props)
+    this.state = {
```

Comment:
I put it into the local state based off of previous conversations of moving too much to the store. I think that this is a good case for local state, because the selected option will only be fully realized when the user is finished with the screen, so we can simplify the props on this component from having two more values (poorly named here) `selectedState` and `onSelected`. Instead we just have the smart component care about the end result, which is the only time anything outside the component will need it.